Bottom: a64e461b027be5e36afb51cc2853d3a2c1257553
Top:    630e8e261421ae859f3de13859ecffc1278970be
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-24 22:35:40 +0100

Revert "hid/gtk: Don't render of soldermask outside board area"

This reverts commit 8ac8d6c6403bd8f322eb146ebe129718cc989a0d.


---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index bf05b6b..c04f975 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -741,11 +741,6 @@ redraw_region (GdkRectangle *rect)
   region.Y2 = MAX(Py(priv->clip_rect.y),
                   Py(priv->clip_rect.y + priv->clip_rect.height + 1));
 
-  region.X1 = MAX (0, MIN (PCB->MaxWidth,  region.X1));
-  region.X2 = MAX (0, MIN (PCB->MaxWidth,  region.X2));
-  region.Y1 = MAX (0, MIN (PCB->MaxHeight, region.Y1));
-  region.Y2 = MAX (0, MIN (PCB->MaxHeight, region.Y2));
-
   eleft = Vx (0);
   eright = Vx (PCB->MaxWidth);
   etop = Vy (0);
@@ -1300,12 +1295,6 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
   region.Y1 = MIN(Py(0), Py(gport->height + 1));
   region.X2 = MAX(Px(0), Px(gport->width + 1));
   region.Y2 = MAX(Py(0), Py(gport->height + 1));
-
-  region.X1 = MAX (0, MIN (PCB->MaxWidth,  region.X1));
-  region.X2 = MAX (0, MIN (PCB->MaxWidth,  region.X2));
-  region.Y1 = MAX (0, MIN (PCB->MaxHeight, region.Y1));
-  region.Y2 = MAX (0, MIN (PCB->MaxHeight, region.Y2));
-
   hid_expose_callback (&ghid_hid, &region, NULL);
 
   gport->drawable = save_drawable;
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 8c1e8a2..23fa8c6 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -961,11 +961,6 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   region.Y1 = MIN (Py (ev->area.y), Py (ev->area.y + ev->area.height + 1));
   region.Y2 = MAX (Py (ev->area.y), Py (ev->area.y + ev->area.height + 1));
 
-  region.X1 = MAX (0, MIN (PCB->MaxWidth,  region.X1));
-  region.X2 = MAX (0, MIN (PCB->MaxWidth,  region.X2));
-  region.Y1 = MAX (0, MIN (PCB->MaxHeight, region.Y1));
-  region.Y2 = MAX (0, MIN (PCB->MaxHeight, region.Y2));
-
   glColor3f (port->bg_color.red / 65535.,
              port->bg_color.green / 65535.,
              port->bg_color.blue / 65535.);
@@ -1214,17 +1209,10 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
                              -gport->view_x0,
                 ghid_flip_y ? gport->view_y0 - PCB->MaxHeight :
                              -gport->view_y0, 0);
-
   region.X1 = MIN(Px(0), Px(gport->width + 1));
   region.Y1 = MIN(Py(0), Py(gport->height + 1));
   region.X2 = MAX(Px(0), Px(gport->width + 1));
   region.Y2 = MAX(Py(0), Py(gport->height + 1));
-
-  region.X1 = MAX (0, MIN (PCB->MaxWidth,  region.X1));
-  region.X2 = MAX (0, MIN (PCB->MaxWidth,  region.X2));
-  region.Y1 = MAX (0, MIN (PCB->MaxHeight, region.Y1));
-  region.Y2 = MAX (0, MIN (PCB->MaxHeight, region.Y2));
-
   hid_expose_callback (&ghid_hid, &region, NULL);
   hidgl_flush_triangles (&buffer);
   glPopMatrix ();
